en/rocky/8/admin_guide/01-presentation.md

```diff
@@ -0,0 +1,299 @@
+# Introduction to the Linux operating system
+
+In this chapter you will learn about GNU/Linux distributions.
+
+****
+**Objectives** : In this chapter you will learn how to:
+
+:heavy_check_mark: Describe the features and possible architectures of an operating system;\
+:heavy_check_mark: Recount the history of UNIX and GNU/Linux;\
+:heavy_check_mark: Choose the right Linux distribution for your needs;\
+:heavy_check_mark: Explain the philosophy of free and opensource software;\
+:heavy_check_mark: Discover the usefulness of the SHELL.\
+
+
+To explain your goals, you should only use action verbs. Because goals need to be quantifiable, measurable and verifiable.
+At the end of the document, you can add questions directly related to these objectives to validate the understanding of the subject.
+
+:checkered_flag: **generalities**, **linux**, **distributions**
+
+**Knowledge**: :star: \
+**Complexity**: :star: \
+**Reading time**: 10 minutes
+****
+
+## What is an operating system?
+
+Linux is an **operating system**.
+
+:clipboard: An operating system is a **set of programs that manages the available resources of a computer**.
+
+Among this management of resources, the operating system has to :
+
+* Manage the physical or virtual memory.
+  * The **physical memory** is made up of the RAM bars and the processor cache memory, which is used for the execution of programs.
+  * The **virtual memory** is a location on the hard disk (the **swap** partition) that allows to unload the physical memory and to save the current state of the system during the electrical shutdown of the computer (system hibernation).
```

Comment:
I'm not sure I would include "hibernation" or suspension as shutdown. I think I'd just put a period at the end of "computer." and leave the rest out.

Reply:
Like this?
```suggestion
  * The **virtual memory** is a location on the hard disk (the **swap** partition) that allows to unload the physical memory and to save the current state of the system during the electrical shutdown of the computer.
```